wiki/Beatmap_Editor/Test_mode/id.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - editor
+  - beatmapping
+  - mapping
+---
+
+# Test mode
+
+![Screenshot tampilan test mode](img/test-mode-example.jpg "Menggunakan test mode dengan autoplay")
+
+**Test mode** adalah sebuah fitur dari [beatmap editor](/wiki/Beatmap_Editor) yang mengizinkan pembuat beatmap untuk memainkan [beatmap](/wiki/Beatmap) sendiri mereka dari beatmap editor untuk dicoba. Mode ini bisi diakses melalui tombol `Test` di bawah-kanan editor.
+
+![Tombol test mode](img/test-mode-button.png "Tombol test mode di beatmap editor")
+
+*Perhatian: Memainkan sebuah beatmap di test mode telah diketahui untuk menyebabkan lag atau framedrop untuk beberapa pengguna.*
+
+Memainkan sebuah beatmap di test mode berbeda dengan memainkannya secara normal, test mode tidak akan mengirim score, tidak menampilkan papan peringkat, tidak akan membiarkan pemain gagal, dan selalu akan menampilkan informasi berikut ini di bagian kiri pada layar:
+
+- [Timestamp](/wiki/Modding/Timestamp) beatmap sekarang.
+- [Muatan SB](/wiki/Beatmap_editor/SB_load).
+- Total jumlah 300, 100, 50, dan miss yang di dapatkan.
+- Persentase dari hit pada waktu yang salah.
+- Delay rata-rata (dalam hitungan milidetik) hit terlambat pada waktu yang salah.
+- [Star rating](/wiki/Beatmapping/Star_rating) beatmap.
+- Total durasi beatmap (dalam hitungan detik).
+- Kombo maksimum yang dapat diraih oleh pemain pada beatmap.
+- Berbagai-bagai tombol shortcut.
+
+Test mode juga berbeda dari bermain seperti biasa karena test mode membutuhkan pengguna untuk menyimpan beatmap mereka terlebih-dahulu sebelum bermain, dan mengizinkan pengguna untuk menonton beatmap mereka dalam "autoplay", yang berfungsi sama seperti mod [Auto](/wiki/Game_modifier/Auto).
```

Comment:
rather concerned about the usage of the many commas in a row here hmm

perhaps try sth like : `terlebih dahulu sebelum bermain. Di samping itu, test mode juga mengizinkan...`

Reply:
sure